auth/src/main/java/com/firebase/ui/auth/util/data/PrivacyDisclosureUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.firebase.ui.auth.util.data;
+
+import android.content.Context;
+import android.support.annotation.RestrictTo;
+import android.support.annotation.StringRes;
+import android.widget.TextView;
+
+import com.firebase.ui.auth.R;
+import com.firebase.ui.auth.data.model.FlowParameters;
+import com.firebase.ui.auth.util.ui.PreambleHandler;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class PrivacyDisclosureUtils {
+
+    private static final int NO_TOS_OR_PP = -1;
+
+    public static void setupTermsOfServiceAndPrivacyPolicyText(Context context,
+                                                               FlowParameters flowParameters,
+                                                               TextView termsText) {
+        PreambleHandler.setup(context,
+                flowParameters,
+                getGlobalTermsStringResource(flowParameters),
+                termsText);
+    }
+
+    public static void setupTermsOfServiceFooter(Context context,
+                                                 FlowParameters flowParameters,
+                                                 TextView footerText) {
+        PreambleHandler.setup(context,
+                flowParameters,
+                getGlobalTermsFooterStringResource(flowParameters),
+                footerText);
+    }
+
+    public static void setupTermsOfServiceAndPrivacyPolicySmsText(Context context,
+                                                                  FlowParameters flowParameters,
+                                                                  TextView termsText) {
+        PreambleHandler.setup(context,
+                flowParameters,
+                R.string.fui_verify_phone_number,
+                getTermsSmsStringResource(flowParameters),
+                termsText);
+    }
+
+    @StringRes
+    private static int getGlobalTermsStringResource(FlowParameters flowParameters) {
+        boolean termsOfServiceUrlProvided = flowParameters.isTermsOfServiceUrlProvided();
+        boolean privacyPolicyUrlProvided = flowParameters.isPrivacyPolicyUrlProvided();
+        if (termsOfServiceUrlProvided && privacyPolicyUrlProvided) {
```

Comment:
Can we add a new line before and after the if/else here and in the other methods. Makes it more readable IMO.

Reply:
Done.